src/webview/pipeline/app/widgets/inputwidget.ts

```diff
@@ -0,0 +1,84 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { createDiv } from '../utils/util';
+import { BaseWidget } from './widget';
+import { PipelineStart } from '../utils/types';
+import { TknResourceType } from '../utils/const';
+import { collectParameterData, collectResourceData } from '../utils/resource';
+import { disableButton } from '..';
```

Comment:
This import looks suspicious to me, please import proper file, not entire directory